### ISSUE TITLE
[curvefs]tools-v2: Add the command of statistical cluster size information

### ISSUE THREAD

**Issue by Cyber-SiKu** [opencurve/curve#2532]
**Is your feature request related to a problem? (你需要的功能是否与某个问题有关?)**

It is necessary to query more accurate cluster scale information, such as total capacity, used capacity, number of files, number of nodes, and cluster running time. You can not complete these problems in one pr, but do a function separately and submit them in stages.

The capacity of the s3 file system can be represented by the quota of the bucket, if not represented as unlimited.

**Describe the solution you'd like (描述你期望的解决方法)**

```bash
$ curve fs list space
```

 |total|used|free|fileNum|inodeNum|running time|
|---|---|---|---|---|---|
|1.0G|1.0G|0.0G|1000|1000|100d
|

**Describe alternatives you've considered (描述你想到的折衷方案)**

**Additional context/screenshots (更多上下文/截图)**

**Comment by Cyber-SiKu** [comment]
> Hi, can I try this issue?

welcome!

**Comment by Cyber-SiKu** [comment]
@WWWindrunner No, this function is brand new, mainly to obtain some fs (data metadata, etc.) usage and other related information.

**Comment by Cyber-SiKu** [comment]
https://github.com/opencurve/curve/blob/master/docs/images/curve-wechat.jpeg

**Comment by lim-yoona** [comment]
Hello, I am a graduate student in the University of the Chinese Academy of Sciences. My major is cyberspace security. I am familiar with programming languages such as Golang, Java and C++. I am interested in technology fields such as distributed systems, cloud native and storage systems. I am good at technology stacks such as Golang server development, storage system principles, etc. 
I have read [tools-v2/docs/zh/develop.md](https://github.com/opencurve/curve/blob/master/tools-v2/docs/zh/develop.md) and [docs/cn/curve工具.md](https://github.com/opencurve/curve/blob/master/docs/cn/curve%E5%B7%A5%E5%85%B7.md), understanding the structure and development method of the curve tool. I think I can complete this issue.
